Application/Program.cs

```diff
@@ -0,0 +1,175 @@
+﻿using Simple_Inventory_Management_System.Domain;
+using Simple_Inventory_Management_System.Services;
+
+class Program
+{
+    static void AddNewProduct(InventoryService inventory)
+    {
+        string name ="";
+        
+        while (true)
+        {
+            Console.Write("Enter product name: ");
+            name = Console.ReadLine();
+            if (name!=string.Empty)
+                break;
+            Console.WriteLine("Please enter the name of product.");
+        }
+
+        double price;
+        while (true)
+        {
+            Console.Write("Enter product price: ");
+            if (double.TryParse(Console.ReadLine(), out price) && price >= 0)
+                break;
+            Console.WriteLine("Invalid input! Please enter a number.");
+        }
+
+        int quantity;
+        while (true)
+        {
+            Console.Write("Enter product quantity: ");
+            if (int.TryParse(Console.ReadLine(), out quantity) && quantity >= 0)
+                break;
+            Console.WriteLine("Invalid input! Please enter a number.");
+        }
+
+        
+        inventory.AddProduct(name, price, quantity);
+        Console.WriteLine("Product is added successfully.");
+    }
+    static void ViewProducts(InventoryService inventory)
+    {
+        if (inventory.InventoryProp.Products.Count > 0)
```

Comment:
```suggestion
        if (inventory.InventoryProp.Products.Any())
```


Use Any() when you only need to check if a collection is non-empty. It is more efficient, especially for IEnumerable<T> types.